the-collab-lab/tcl-66-smart-shopping-list
# 25. Make app more responsive for mobile compatibility

## Description

- Updated the sizing of components based on screen size
- Updated Logo
- Updated Login page styling

## Related Issue

closes #55 

## Acceptance Criteria

- [x] App looks good for mobile sizing
- [x] Slide in/out Nav bar

## Type of Changes

enhancement, accessibility

## Updates

### Before
<img width="929" alt="Screenshot 2024-04-02 at 10 06 56 PM" src="https://github.com/the-collab-lab/tcl-66-smart-shopping-list/assets/98231995/6fdbef9b-5f1c-4bc3-8111-a288b5ffc6dc">

https://github.com/the-collab-lab/tcl-66-smart-shopping-list/assets/98231995/34324c6b-95db-4814-96ee-2edbf9223027


### After

<img width="930" alt="Screenshot 2024-04-02 at 10 06 42 PM" src="https://github.com/the-collab-lab/tcl-66-smart-shopping-list/assets/98231995/b09d72f4-28eb-4b06-87f8-33e783d11d71">

https://github.com/the-collab-lab/tcl-66-smart-shopping-list/assets/98231995/340eec9c-ff96-4c8b-9fd0-5b10f1c3482b


## Testing Steps / QA Criteria

1. git pull from main
2. git checkout an-issue-25
3. inspect page and check responsiveness of different devices
